### PR TITLE
Playlist modal

### PR DESCRIPTION
**Pull request by Vindexus**

### CODE REVIEW

**Comment by tmathews** [comment]
Animated screenshot required.

**Comment by Vindexus** [comment]
playlistmodals

**Comment by tmathews** [comment]
Can we just make the actual modal scroll within the shadow box?

File: src/css/layout.css [hunk 1]
```diff
@@ -844,11 +848,12 @@ span.note a {
   width: 90%;
   max-width: 1000px;
   position: relative;
-  margin: 50px auto;
+  margin: 4% auto;
```
Comment (tmathews) [inline src/css/layout.css]: Unrelated

File: src/js/notices.js [hunk 2]
```diff
@@ -124,7 +124,7 @@ function clickLinkNoticeLink (e, name) {
 =            COMPLETE PROFILE            =
 ========================================*/
 
-var completeProfileNotice = new SiteNotice({
+siteNotices.completeProfileNotice = new SiteNotice({
```
Comment (tmathews) [inline src/js/notices.js]: Unrelated

File: src/css/layout.css [hunk 3]
```diff
@@ -844,11 +848,12 @@ span.note a {
...
 }
 
 .modal > .panel:first-of-type {
   margin-top: 0px;
+  margin-bottom: 0px;
```
Comment (tmathews) [inline src/css/layout.css]: Unrelated
